content/guides/compute.md

```diff
@@ -4,3 +4,331 @@ title: Compute Engine documentation | ARGO
+`T1_JobA_ap.json`
+
+The information in the availability profile json file is automatically picked up by the compute-engine during computations. 
+
+###### Contents of an availability profile json file
+
+- `"name": "string"`  
+the name of the availability profile  
+- `"namespace": "string"`  
+the name of the namespace used by the profile  
+- `"metric_profile": "string"`  
+the name of the metric profile linked to this availability profile  
+- `"metric_ops": "string"`  
+the default operation to be used when aggregating low level metric statuses
+- `"group_type": "string"`  
+the default endpoint group type used in aggregation
+
+In the availability profile json file also are declared custom grouping of services to be used in the aggregation. The grouping of services are expressed in the json "groups" list see example below:
+
+```
+"groups": {
+    "my_group_of_services_1": {
+      "services":{
+        "service_type_A":"OR",
+        "service_type_B":"OR"
+        },
+       "operation":"OR"
+    },
+    "my_group_of_services_2": {
+      "services":{
+        "service_type_C":"OR",
+        "service_type_D":"OR"
+        },
+       "operation":"OR"
+    },
+"operation":"AND"
+```
+
+In the above example the service types are grouped in two groups: ***my_group_of_services_1*** and ***my_group_of_services_2***. Each group contains a "service" list containing service types included in the group as fields and the operation values in orded to choose who to aggregate the various instances of a specific service. For example if for ***"service_type_A"*** are 3 service endpoints available they are goint to be aggregated using the OR operation. The ***"operation"*** field under each group of services is used to declare the operation that will be used to aggregate the service types under that group.   
+The outer ***"operation"*** field in the root of the json document is used to declare the operation used to aggregate the various groups in order to produce the final endpoint aggregation result.
+
+
+
+
+
+
+
+### Executable Scripts of the standalone edition
+In the folder `/usr/libexec/ar-compute/standalone/` reside executable scripts that can be used for uploading metric data and sync data to the hadoop cluster (HDFS Filesystem). 
+##### upload_metric.py 
+The specific script is used in order to upload daily metric data (relative to a tenant) to HDFS.   
+full path: `/usr/libexec/ar-compute/standalone/upload_metric.py`
+
+parameters:
+- `-d --date {YYYY-MM-DD}`  
+specifies the date of the metric data we want to upload
+- `-t --tenant {STRING}`  
+a case-sensitive string specifing the name of the tenant
+
+
+##### upload_sync.py
+The specific script is used in order to upload daily sync data (relative to a tenant and a job) to HDFS.
+full path: `/usr/libexec/ar-compute/standalone/upload_sync.py` 
+
+parameters:
+- `d --date {YYYY-MM-DD}`  
```

Comment:
It should be `-d` instead of `d`. Please check also the other parameters whether they are missing the `-`